star-wars-browser/src/store/films/reducer.js

```diff
@@ -0,0 +1,13 @@
+import { createReducer } from '@reduxjs/toolkit';
+import { initialState } from './defaultState';
+import { setFilms } from './actions';
+
+const filmReducer = createReducer(initialState, (builder) => {
+  builder
+    .addCase(setFilms, (state, action) => {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
możesz na początku pliku dać
```
/* eslint-disable no-param-reassign */
```
bo to się będzie powtarzać za każdym razem

Reply:
poprawione